crates/tap-agent/src/bin/mock_tap_aggregator_server_runner.rs

```diff
@@ -0,0 +1,74 @@
+use std::net::SocketAddr;
+
+use tap_aggregator::grpc::{
+    tap_aggregator_server::{TapAggregator, TapAggregatorServer},
+    RavRequest, RavResponse,
+};
+use tokio::sync::oneshot;
+use tonic::{transport::Server, Request, Response, Status};
+
+#[derive(Default)]
+pub struct MockTapAggregatorService;
+
+#[tonic::async_trait]
+impl TapAggregator for MockTapAggregatorService {
+    async fn aggregate_receipts(
+        &self,
+        _request: Request<RavRequest>,
+    ) -> Result<Response<RavResponse>, Status> {
+        // Do nothing, just return an unimplemented response
+        Err(Status::unimplemented("This is a mock service"))
```

Comment:
This whole file for me doesn't make sense, you can just use the same thing that we are using for older tests: `tap_aggregator::server::run_server`